docs/strings.md

```diff
@@ -139,6 +139,7 @@ For example, when parsed, the following string would represent the date **28 Iya
 The syntax of the calendar suffix is currently on a standards track led by the [IETF SEDATE](https://datatracker.ietf.org/wg/sedate/about/) working group.
 
 The calendar identifiers [are defined by CLDR](http://unicode.org/reports/tr35/#UnicodeCalendarIdentifier) as [a sequence of 3-8 character BCP47 subtags](http://unicode.org/reports/tr35/#unicode_locale_extensions).
+Usually there's only one subtag in the sequence, although Islamic calendar variations currently use two subtags.
```

Comment:
I don't believe this is necessary, and this might not hold true at some point in the future as there are more additions to the CLDR dataset.

Reply:
OK I changed to this text below. Better?
 
> Usually there's only one subtag in the sequence, although some calendars may use more.

My goal was to pre-empt the complaint from users who say "why are some calendar IDs 16 chars long when you said they were 2-8 chars?" 

Even though the current text is correct because it says "sequence of...subtags", readers who are skimming or who are non-fluent English readers may think "2-8 character" applies to the calendar ID as a whole instead of each subtag.  Seemed worth a clarification.